orc8r/cloud/go/models/swagger-common.yml

```diff
@@ -151,3 +153,23 @@ definitions:
       message:
         type: string
         example: Error string
+  registration_info:
+    type: object
+    required:
+      - registration_token
+      - root_ca
+      - domain_name
+    properties:
+      registration_token:
+        description: registration_token is a token for the operator to give to the AGW; it keys to logical and network ID
+        type: string
+        minLength: 4
+        example: reg_a9vnap30fN0anrfjVneB
+      root_ca:
+        description: root_ca is a certificate that access gateways (AGW) can use to handshake and communicate with the stated orc8r
+        type: string
+        example: =example rootCA.pem=
+      domain_name:
+        description: domain_name is the domain name where the stated orc8r can be accessed
+        type: string
+        example: example.com
```

Comment:
This seems a little strange to me - wouldn't this be the same for every gateway? Why is this configurable per gateway?

Reply:
it is returned by the orc8r to make registering from the device faster -- not configurable